build.sh

```diff
@@ -5,8 +5,15 @@ cd build
 # Run cmake to generate make files
 cmake ..
 
-# Run make to build the project. To build utilizing multiple cores, append `-j` and the amount of cores to utilize, for example `make -j8`
-make
+# Windows build
+if [[ "$OSTYPE" =~ ^msys ]]; then
+
+	cmake --build . --config Release
```

Comment:
We do not build in release for our binaries.  We build in RelWithDebInfo.  Why exactly does make not work here?  This seems pointless at the moment

Reply:
I was just following what was on the README. For windows it says to use cmake instead of make. I can look into using make instead, if the README is outdated

Reply:
Try using make instead if possible.

Reply:
make looks to be a gnu exclusive command.

Reply:
Instead of hard coding a check for windows I think it'll be more ideal to make a version of this script that doesn't need any checks or different commands for different OS's since we use cmake.

Reply:
So is the plan then to change from using `make` to using `cmake --build . --config Release` for all systems?

Reply:
for the build script yes - keep it consistent across platforms so there is uniformity (and so we dont need to check for os type)